pkg/ddl/partition.go

```diff
@@ -337,19 +347,15 @@ func updateAddingPartitionInfo(partitionInfo *model.PartitionInfo, tblInfo *mode
 }
 
 // rollbackAddingPartitionInfo remove the `addingDefinitions` in the tableInfo.
-func rollbackAddingPartitionInfo(tblInfo *model.TableInfo) ([]int64, []string, []*placement.Bundle) {
+func rollbackAddingPartitionInfo(tblInfo *model.TableInfo) ([]int64, []string) {
```

Comment:
I think this function should be renamed to `removePartitionAddingDefinitionsFromTableInfo` now ?